docs/tutorials/ruby-driver-authentication.txt

```diff
@@ -101,6 +101,13 @@ derived from the distinguished subject name of this certificate.
 This authentication method requires the use of SSL connections with
 certificate validation.
 
+To authenticate the client, you will need a valid SSL certificate
+and private encryption key. These can be stored in separate files,
+or together in one file (in the PEM format). Even if the certificate
+and private key are stored in the same file, you must specify the path to
+that file by passing both the ``ssl_cert`` and ``ssl_key`` options
+to the client.
```

Comment:
I'd love feedback on this paragraph! Is it clear enough? Should I talk about how certificate files are generated, or is that not relevant?

Reply:
I think it is well written as it is.